modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,362 @@
+<?php declare(strict_types=1);
+/**
+ * Get Electrophysiology Session Data.
```

Comment:
new modules shouldn't be using the `ajax` directory at all, this should be converted to a real URL endpoint now that LORIS has a way to route requests properly. It should be something like `$lorisBase/electrophysiology_browser/eeg_session/$sessionid` (for some reasonable unique definition of `$sessionid`)